index.js

```diff
@@ -84,10 +90,12 @@ module.exports = {
             HAS_NATIVE_COMPUTED_GETTERS: gte(trueEmberVersion, '3.1.0-beta.1'),
 
             IS_GLIMMER_2: gte(trueEmberVersion, '2.10.0'),
+            IS_RECORD_DATA: gte(emberDataVersion, '3.5.0'),
```

Comment:
Why `IS_RECORD_DATA` here but `SUPPORTS_MODEL_DATA` below? Are there actually two different features? I thought it was the same feature which started as "model data" but was renamed to "record data"?

Reply:
see notes in the PR but TL;DR is the feature started it's life as `ModelData` and was implemented as such. The differentiation is important as hook names change.

Reply:
The feature was released in 3.5.0 (for the first time) under record data, I don't think we should have two flags here...